src/libraries/Native/Unix/System.Native/pal_interfaceaddresses.c

```diff
@@ -114,7 +114,7 @@ int32_t SystemNative_EnumerateInterfaceAddresses(IPv4AddressFound onIpv4Found,
             freeifaddrs(headAddr);
```

Comment:
@dotnet/ncl Could you please review the networking related changes?

Reply:
`__u32 ethtool_cmd_speed()`  function is not defined in NDK's header, but `struct ethtool_cmd` has `__u16 speed` and `__u16 speed_hi` defined there.

I have ifdef'd it for Android and just used `.speed`, ignoring the `.speed_hi` as done by Linux distros https://github.com/torvalds/linux/blob/bd2463a/include/uapi/linux/ethtool.h#L125. Not sure if there is any down side to this approach on Android.

That's the main change in networking code for Android.